apps/dashboard/src/app/(dashboard)/dashboard/connect/ecosystem/[slug]/(active)/hooks/use-partners.ts

```diff
@@ -0,0 +1,31 @@
+import { useLoggedInUser } from "@3rdweb-sdk/react/hooks/useLoggedInUser";
+import { useQuery } from "@tanstack/react-query";
+import type { Ecosystem, Partner } from "../../../types";
+
+export function usePartners({ ecosystem }: { ecosystem: Ecosystem }) {
+  const { isLoggedIn } = useLoggedInUser();
+
+  const partnersQuery = useQuery({
+    queryKey: ["partners", ecosystem.id],
```

Comment:
ideally we want left lower specificity, right higher specificity of the query:


```suggestion
    queryKey: ["ecosystem", ecosystem.id, "partners"],
```